authui/src/authflowv2/components/screen-action-layout.css

```diff
@@ -0,0 +1,56 @@
+@layer components {
+  .screen-action-layout {
+    /* Expected layout:
+        <header>
+          <title/>
+          <description/>
+        </header>
+        <flex-1 spacing/>
+        <footer>
+          <button/>
+        </footer>
```

Comment:
The structure of the elements in comment is different from the actual usage.
In wechat.html, `<title>` and `<description>` are not wrapped by another element.